addons/calendar/models/calendar_recurrence.py

```diff
@@ -131,7 +131,7 @@ class RecurrenceRule(models.Model):
     def _compute_name(self):
         for recurrence in self:
             period = dict(RRULE_TYPE_SELECTION)[recurrence.rrule_type]
-            every = _("Every %(count)s %(period)s, ", count=recurrence.interval, period=period)
+            every = _("Every %(count)s %(period)s", count=recurrence.interval, period=period)
```

Comment:
@arj-odoo  Since you changed the string in `_()`, I am not sure if the translation will be affected.